### ISSUE TITLE
Warning: Moving average ignored a value of inf

### ISSUE THREAD

**Issue by zeroRains** [haotian-liu/yolact_edge#37]
Hi!I have a problem in train.py. I used the model you provide to train.It can be right at the begining ,but it will 
be bad like this .
![image](https://user-images.githubusercontent.com/57100978/105625594-4ce95500-5e65-11eb-8fd8-45ab1b2cb806.png)

I use the coco to train but the data look so strange.
What's wrong abaout this train?

**Comment by cyrilzakka** [comment]
First make sure your image size is a multiple of 32, then decrease your learning rate (e.g. 5e-4) and increase your batch size to something greater than 1.

**Comment by sdimantsd** [comment]
@cyrilzakka Why the image size should by multiple of 32?

**Comment by zeroRains** [comment]
I have already changed the min_size and max_size as 320 in congfig.py, lr have alread been set as 5e-4 and the batch size is 8. But it still show the warning and the data look so strange.
A question
where to set the epoch in a train?

**Comment by zeroRains** [comment]
I have already changed the min_size and max_size as 320 in congfig.py, lr have alread been set as 5e-4 and the batch size is 8. But it still show the warning and the data look so strange.
A question
where to set the epoch in a train?

**Comment by cyrilzakka** [comment]
@zeroRains here is the line where you should change the image size: https://github.com/haotian-liu/yolact_edge/blob/662d760f8b2d8b4409d385aaf172e155aaa3a3d8/data/config.py#L745

**Comment by haotian-liu** [comment]
@cyrilzakka Thank you for helping!
@zeroRains This is usually due to the learning rate not suitable for the dataset and the config you are using. Also, when you are changing the image size, there are some other things to change (more than `max_size`):

1. prediction anchor priors: `cfg.backbone.pred_scales`.
2. size of discarded detection: `cfg.discard_box_width`,`cfg.discard_box_height`

**Comment by zeroRains** [comment]
Thank you for your help! And the problem have been solve but and couldn't find where can I set the epoch and I also want to ask why the image size should by mutiple of 32?

**Comment by haotian-liu** [comment]
@zeroRains 
1. You can set the number of iterations using `cfg.max_iter`.
2. It does not necessarily to be a multiple of 32: the default image size of YolactEdge is 550.

**Comment by zeroRains** [comment]
Thank you for you help! I have not problem.
